### ISSUE TITLE
is it possible to generate charts based on jmx values ?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javamelody/javamelody#384]
```
Is it possible to generate charts based on jmx values ?

it would automatically solve 
https://code.google.com/p/javamelody/issues/detail?id=224&colspec=ID%20Type%20St
atus%20Priority%20Owner%20Summary
and give more possibilities.

Is it hard or easy to implement ?

Do You consider implementing such functionality ?

BR
Jakub
```

Original issue reported on code.google.com by `jjaku...@gmail.com` on 20 Feb 2014 at 10:10

**Comment by sanketmeghani** [comment]
@evernat Is this implemented and integrated into JavaMelody? We need to add custom counters/metrics and related graphs. What is the best way to add custom metrics/counters without updating JavaMelody source code in latest JavaMelody release.

**Comment by sanketmeghani** [comment]
If this has not been implemented and you think it could be useful, may be I can try to implement it. In that case is there a possibility I can see above pull request https://github.com/evernat/javamelody/pull/2. 

I am trying out the options listed https://github.com/javamelody/javamelody/wiki/UserGuide#9-business-facades-if-spring to see if we can use to implement custom counters/metrics.

**Comment by evernat** [comment]
@sanketmeghani 
Yes it could be useful. PR is now at https://github.com/evernat/javamelody-mirror-backup/pull/2. See comments there.
